tests/queries/0_stateless/02992_all_columns_should_have_comment.sql

```diff
@@ -1,4 +1,8 @@
 SYSTEM FLUSH LOGS;
 SELECT 'Column ' || name || ' from table ' || concat(database, '.', table) || ' should have a comment'
 FROM system.columns
-WHERE (database = 'system') AND (comment = '') AND (table NOT ILIKE '%_log_%') AND (table NOT IN ('numbers', 'numbers_mt', 'one', 'generate_series', 'generateSeries', 'coverage_log')) AND (default_kind != 'ALIAS');
+WHERE (database = 'system') AND
+      (comment = '') AND
+      (table NOT ILIKE '%_log_%') AND
```

Comment:
FYI, `_` is a wildcard for LIKE.
```suggestion
      (table NOT ILIKE '%\_log\_%') AND
```